### PR TITLE
Lilypad: Remove psx/ps1 emulator compatibility

### PR DESCRIPTION
**Pull request by lightningterror**
ps1 games will still work fine under pcsx2, just to clarify.

Same as https://github.com/PCSX2/pcsx2/pull/3274

### CODE REVIEW

**Comment by orbea** [comment]
I'm just curious, can you elaborate on what exactly is being removed in this PR?

**Comment by KrossX** [comment]
Compatibility with the plugin specs for PS1 emulators, and all the workarounds for PS1 emulators as well.

**Review by GovanifY — CHANGES_REQUESTED** [review]
There is some mentions of PS1 mouse that aren't removed in Lilypad.cpp, Tooltips.cpp, Config.cpp and resource.h

File: plugins/LilyPad/Config.cpp [hunk 1]
```diff
@@ -42,38 +42,8 @@ const wchar_t *padTypes[] = {
     L"PS1 Mouse",
```
Comment (GovanifY) [inline plugins/LilyPad/Config.cpp:42]: Eg a reference there

**Comment by RedDevilus** [comment]
Summarised:
Config.cpp

L2133-2134
```
else if (config.padConfigs[port][slot].type == MousePad)
                psp.pszTemplate = MAKEINTRESOURCE(IDD_CONFIG_PS1_MOUSE);
```
Lilypad.rc

L325
```
IDD_CONFIG_PS1_MOUSE DIALOGEX 0, 0, 424, 283
```
L531
```
IDD_CONFIG_PS1_MOUSE, DIALOG
```
Lilypad.cpp
Mentions of MODE_PS1_MOUSE
L89
`#define MODE_PS1_MOUSE 0x12`
L798
 `pads[port][slot].mode = MODE_PS1_MOUSE;`
L1207-1208
```
 DEBUG_OUT(MODE_PS1_MOUSE);
                        return MODE_PS1_MOUSE;
```
Resource.h
L15
`#define IDD_CONFIG_PS1_MOUSE            0x70`

Tooltips.cpp
L54-55
```
  L"\"PS1 Mouse\" emulates the Playstation Mouse. This controller can only be used in a number of PS1 games like \"Command & Conquer: Red Alert\" and \"Myst\".\n\n"
                   L"\"neGcon\" emulates a controller that can be used in a number of PS1 games and PS2 games like the \"Ridge Racer\" and \"Ace Combat\" series.";
```

**Pull request merged**